.flake8

```diff
@@ -20,7 +20,7 @@ per-file-ignores =
     dpctl/_sycl_platform.pyx: E999, E225
     dpctl/_sycl_queue.pyx: E999, E225, E226, E227
     dpctl/_sycl_queue_manager.pyx: E999, E225
-    dpctl/memory/_memory.pyx: E999, E225, E226, E227
+    dpctl/memory/_memory.pyx: E999, E225, E226, E227, E125
```

Comment:
Where is it complaining about E125?

Reply:
Alignment

Reply:
Yes, but where. I am asking since perhaps we can fix the issue instead of suppressing ti.

Reply:
in `_memory.pyx`, lines 438-440

Reply:
Try moving the closing parenthesis on L439 to the next line aligned with the `cdef`.

Reply:
That worked! All concerns have been addressed.